.github/workflows/pr.yml

```diff
@@ -0,0 +1,105 @@
+name: "Pull Request Build"
+on:
+  workflow_dispatch:
+  pull_request:
+jobs:
+  linux-build-local:
+    name: PR - Linux - JVM ${{ matrix.java }} - Local
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8 ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Cache .m2 registry
```

Comment:
Can't you do that with the setup-java action now?

Reply:
Didn't know we can cache the Maven dependencies using the `setup-java` action. Updated.